.github/workflows/lint.yml

```diff
@@ -6,9 +6,6 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8"]
 
     name: Python ${{ matrix.python-version }}
```

Comment:
```suggestion
    name: Python 3.8
```

Reply:
Let's just call it "Lint". Will update, thanks!